monitoring/monitors/viya/podMonitor-sas-go-pods.yaml

```diff
@@ -22,11 +22,32 @@ spec:
     #   regex: golang
```

Comment:
Take a look at the commented out lines on 19-23: would be good to determine whether most/all Go microservices are now using the correct value for that annotation...and then uncomment out the lines if appropriate.

Reply:
Yeah, I just checked it out and that wasn't a change that I had made.  If it's already been implemented, I'll delete the comment.